### PR TITLE
change nestjs to 8

### PR DESCRIPTION
**Pull request by KenChanSwivel**

### CODE REVIEW

**Comment by KenChanSwivel** [comment]
Please upgrade nest js version so the one who are using nestjs 8 can be no problem. I have tested on our side.

**Comment by valentinarodrigues** [comment]
Facing issue with this package when upgrading to nestjs 8. Can we please merge this?

File: package.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+  "name": "@swivel-admin/swivel-nestjs-redis",
   "version": "1.3.3",
   "description": "a NestJS ioRedis module",
-  "author": "skunight",
+  "author": "swivelsoftware",
   "license": "MIT",
   "repository": {
     "type": "git",
-    "url": "https://github.com/skunight/nestjs-redis.git"
+    "url": "https://github.com/swivelsoftware/nestjs-redis.git"
```
Comment (UziTech) [inline package.json:9]: These changes should be removed from this PR

**Review by UziTech — CHANGES_REQUESTED** [review]

**Comment by KenChanSwivel** [comment]
please use this one
https://github.com/skunight/nestjs-redis/pull/104

**Pull request closed without merging**